packages/foreman/tfm/tfm.spec

```diff
@@ -327,6 +327,9 @@ selinuxenabled && load_policy || :
 %{_root_sysconfdir}/rpm/macros.%{scl_name}-scldevel
 
 %changelog
+* Fri Jul 16 2021 Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl>
```

Comment:
Missing the version-release at the end

Reply:
Surprising - I used obal but now I recall that the latest released version doesn't work and you need a git version.